route/bgmetadata.go

```diff
@@ -138,8 +144,29 @@ func NewBgMetadataRoute(key, prefix, sub, regex, aggregationCfg, schemasCfg stri
 	m.rm = metrics.NewRouteMetrics(key, "bg_metadata", nil)
 
 	go m.clearBloomFilter()
+	switch storageName {
+	case "cassandra":
+		m.storage = storage.NewCassandraMetadata()
+		m.maxConcurrentWrites = make(chan int, 1)
+	case "elasticsearch":
+		if v, ok := additionnalCfg.(*cfg.BgMetadataESConfig); ok == true {
+			m.storage = storage.NewBgMetadataElasticSearchConnectorWithDefaults(v)
+			m.maxConcurrentWrites = make(chan int, 10)
```

Comment:
Is it used ?

Reply:
Actually yes in the bgmatadata.dispatch() it acts as a semaphore, it may need to be renamed since the UpdateMetricMetadata for ES will not do concurrent writes but will aggregate the metrics to make a bulk write